### PR TITLE
feat: add 2 projects into Playground (Labs)

### PR DESCRIPTION
**Pull request by oriole-pub**
Playground (Labs): a set of sandbox environments where developers can experiment with Nervos development and test their code.

- project 1 in webcontainer: Project: Inscribe "Hello Common Knowledge Base!" to CKB Testnet
- project 2 with external link: Toss A Coin -- like button with CKB

### `npm run start` in your browser
image

## Snapshots
- image

## References
- https://webcontainers.io/

### CODE REVIEW

File: site/package.json [hunk 1]
```diff
@@ -48,7 +48,26 @@
...
+    "@codemirror/lang-javascript": "^6.0.1",
+    "@codemirror/lang-json": "^6.0.0",
+    "@codemirror/lang-markdown": "^6.0.0",
+    "@codemirror/language": "^6.2.0",
+    "@codemirror/language-data": "^6.1.0",
+    "@codemirror/search": "^6.0.0",
+    "@codemirror/state": "^6.1.0",
+    "@codemirror/theme-one-dark": "^6.0.0",
+    "@codemirror/view": "^6.0.3",
+    "codemirror": "^6.0.1",
```
Comment (Flouse) [inline site/package.json:69]: image

### Does `codemirror` support adjusting the colors of the above code blocks?

Comment (oriole-pub) [reply site/package.json:69]: I know where the problem is, and I will handle it

**Review by Flouse — APPROVED** [review]
image

**Comment by Flouse** [comment]
image

# Code in web container not updating live after file changes

Although files are changed, but the code in the webcontainer is not updated live.

**Pull request merged**